src/js/user-profile/components/FormList.jsx

```diff
@@ -29,9 +29,10 @@ class FormList extends React.Component {
    const { savedForms: forms } = this.props;
    const verifiedSavedForms = forms.filter(isSIPEnabledForm);
    const hasVerifiedSavedForms = !!verifiedSavedForms.length;
+   const savedApplicationsHeader = forms.length > 1 ? 'Applications in process' : 'Application in process';
```

Comment:
Should progress instead of process, I think?

Reply:
@peggygannon had changed this in [this comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4739#issuecomment-328627641) on the issue.